src/test/extension.test.ts

```diff
@@ -237,11 +379,45 @@ const languageCode = [
     },
 ]
 
+const unsupportedLanguageCode = [
+    {
+        lang: 'C++',
+        code: `   {
+            log_error("Exception occurred!");
+            throw;
+          }`,
+    },
+]
+
+const expectedLanguageTestOutcome = [
+    {
+        range: new sourcegraph.Range(new sourcegraph.Position(1, 0), new sourcegraph.Position(1, 0)),
+        isWholeLine: true,
+        after: {
+            backgroundColor: '#f2736d',
+            color: 'rgba(255, 255, 255, 0.8)',
+            contentText: ' View logs in Sentry (❕)» ',
+            hoverMessage: ' Add Sentry projects to your Sentry extension settings for project matching.',
+            linkURL: 'https://sentry.io/organizations/sourcegraph/issues/',
+        },
+    },
+]
+
 describe('decorate Editor', () => {
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
+
     projects[0].patternProperties.lineMatches = []
-    for (const [i, codeExample] of languageCode.entries()) {
-        it('check common pattern matching for ' + languageCode[i].lang, () =>
-            expect(decorateEditor([], codeExample.code)).toEqual([decorationsList[3]])
+    for (const [i, codeExample] of supportedLanguageCode.entries()) {
+        it('check common pattern matching for ' + supportedLanguageCode[i].lang, () =>
```

Comment:
This still matches the test to the expected output by index, which is brittle and hard to review - consider adding an `expected` property here too